### DISCUSSION TITLE
How can I mark all entries on a site as "already translated"?

### DISCUSSION THREAD

**Discussion by chadcrowell** [AcclaroInc/craft-translations#287] [Ideas]
Trying out the plugin on a site with 6 languages. This site's Craft install is a replacement content management system for an old ExpressionEngine 2 install - thus, all the entries already have translations. I'd like to take advantage of the "Modified Source Entries" widget on the dashboard, but the plugin thinks that none of my entries have ever been translated. Is there a tool I can use, or a query I can run on the db, to mark everything as translated from the start?

**Comment by ifrimere** [comment]
@sidedwards - do you have any ideas that can help @chadcrowell with this request. Thanks.
